build.sbt

```diff
@@ -303,17 +339,29 @@ lazy val mtest = project
   .in(file("tests/mtest"))
   .settings(
     skip.in(publish) := true,
-    crossScalaVersions := List(V.scala212, V.scala211),
+    crossScalaVersions := V.supportedScalaVersions,
     libraryDependencies ++= List(
-      "com.geirsson" %% "coursier-small" % "1.3.3",
-      "org.scalameta" %% "testkit" % V.scalameta,
-      "com.lihaoyi" %% "utest" % "0.6.0"
+      "io.get-coursier" %% "coursier" % "2.0.0-RC2-2",
+      "org.scalameta" %% "testkit" % V.scalameta
+    ) ++ crossSetting(
+      scalaVersion.value,
+      if211 = List("com.lihaoyi" %% "utest" % "0.6.8"),
+      otherwise = List("com.lihaoyi" %% "utest" % "0.6.9")
+    ),
+    scalacOptions ++= crossSetting(
+      scalaVersion.value,
+      if211 = List("-Xexperimental", "-Ywarn-unused-import")
+    ),
+    scalacOptions --= crossSetting(
+      scalaVersion.value,
+      if211 = List("-Ywarn-unused:imports")
     ),
     buildInfoPackage := "tests",
     buildInfoObject := "BuildInfoVersions",
     buildInfoKeys := Seq[BuildInfoKey](
       "scala211" -> V.scala211,
-      "scala212" -> V.scala212
+      "scala212" -> V.scala212,
```

Comment:
Missing `scala213`

Reply:
I'll add it for consistency, but it's not used anywhere.